README.md

```diff
@@ -10,8 +10,11 @@ These tutorials use the [reStructuredText](http://www.sphinx-doc.org/en/stable/r
 
 All content in this repository is open source and released under the [BSD License v3](https://opensource.org/licenses/BSD-3-Clause). Each individual source code file should contain a copy of the license.
 
-This repository is currently built automatically by two systems. Travis builds the documentation for Melodic and ROS Build Farm builds the documentation for older versions:
-- [![Travis Status](https://travis-ci.org/ros-planning/moveit_tutorials.svg?branch=master)](https://travis-ci.org/ros-planning/moveit_tutorials) [Github Pages + Travis](https://ros-planning.github.io/moveit_tutorials/): latest
+This repository is currently built automatically by two systems. Github Actions builds the documentation for Noetic and Melodic, and ROS Build Farm builds the documentation for older versions:
```

Comment:
```suggestion
This repository is currently built automatically by two systems. Github Actions builds the documentation for Noetic and Melodic (because they are the same tutorials),
and ROS Build Farm builds the documentation for older versions:
```
Is this statement correct? Or have we configured GActions to build mutiple versions? I'm only aware of the former

Reply:
```
jobs:
  default:
    strategy:
      matrix:
        env:
          - IMAGE: master-source
            CCOV: false
            CATKIN_LINT: true
          - IMAGE: noetic-source
            CCOV: false
            CATKIN_LINT: true
```

Here we specify that the tutorials would be built using melodic and noetic containers seperately. It essentially builds the master branch with two different distros.